### ISSUE TITLE
Report attempt to delete named tuple entry

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#7197]
Pyright doesn't currently report an error for an attempt to delete a named tuple field by name.

```python
class Point(NamedTuple):
    x: int
    y: int
    units: str = "meters"

p = Point(1, 2)

del p.x  # Should generate error
del p[0]  # Correctly generates error
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.350, which I just published.
